genesyscloud/resource_genesyscloud_architect_schedules_test.go

```diff
@@ -0,0 +1,111 @@
+package genesyscloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v53/platformclientv2"
+)
+
+func TestAccResourceArchitectSchedules(t *testing.T) {
```

Comment:
Im getting an error running this locally: `Failed to update schedule CX as Code Schedule: API Error: 400 - Field 'divisionId' is required and cannot be empty.`

Reply:
I've pushed my updates based on your comments. I've addressed everything except for this one as I am unable to recreate this error. :)